core/vm/evm.go

```diff
@@ -102,6 +104,18 @@ type TxContext struct {
 	GasPrice *big.Int       // Provides information for GASPRICE
 }
 
+// AtomicTransactor enables smart contracts to create import and export transactions between the
+// C-Chain and the X-Chain via precompile contracts.
+type AtomicTransactor interface {
```

Comment:
New interface. Currently only has export functionality as I am working out how this should all fit together.